src/renderers/additional/tree-renderer.ts

```diff
@@ -139,15 +139,20 @@ export class TreeMasterDetailRenderer extends Renderer implements DataChangeList
 
     this.appendChild(div);
     this.dialog = document.createElement('dialog');
+    this.dialog.classList.add('jsf-treeMasterDetail-dialog');
     const title = document.createElement('label');
     title.innerText = 'Select the Item to create:';
+    title.classList.add('jsf-treeMasterDetail-dialog-title');
     this.dialog.appendChild(title);
     const dialogContent = document.createElement('div');
     dialogContent.classList.add('content');
+    dialogContent.classList.add('jsf-treeMasterDetail-dialog-content');
     this.dialog.appendChild(dialogContent);
     const dialogClose = document.createElement('button');
     dialogClose.innerText = 'Close';
     dialogClose.onclick = () => this.dialog.close();
+    dialogClose.classList.add('btn');
```

Comment:
No, it's fine. It's just that we have to aggre on a style naming convention. I'd go with `button` in this case as and our tests already make use this.